src/node/async_hooks.ts

```diff
@@ -13,6 +13,22 @@ class AsyncHook {
   public disable(): this {
     return this;
   }
+
+  // IMPORTANT
+  //
+  // The following methods were defined in unenv while they don't exist in Node.js
+  // async_hooks module. This is a bug with unenv.
+  // Unfortunately, unless we are 100% sure these methods are not used in production,
+  // we can not remove them from workerd.
```

Comment:
When you say "cannot remove them" do you mean "cannot not add them"?
They are not currently in workerd but we are going to remove them from unenv.
Is that right?